### PR TITLE
Update operations.html

### PR DESCRIPTION
**Pull request by neptunia**
Fix Pipe characters dropped from Fact during Parsing #2877

## Description

Fixes an issue with facts containing the pipe character |, so that they are no longer dropped. Instead of using a separator value to delimit the fact name and value, prepend and store the length of the fact name, then extract the name and value using that.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

Ran the modified code with links with associated facts. Some facts contained the pipe character and some did not.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by elegantmoose** [comment]
@neptunia @bjeffries Ive patched this issue for v5 (https://github.com/mitre/magma/pull/41) . Do you need it for v4 as well?

TLDR - was attempting to put in a PR against v.4.2 but now hitting other issues so just making sure that the change would be useful in v4 as well before I keep going.
